content/md/products-super-power/improve-data-quality.md

```diff
@@ -58,17 +58,31 @@ To help you improve your products **enrichment** and **consistency**, simply fol
 ![Image data quality panel](../img/data-quality-panel.png)
 From this panel, you will know which attributes need to be improved and how to improve them.
 
+::: info
+Links can lead you either to a specific attribute or to the list of improvable attributes in the attribute panel.
+:::
+
+
+![Image data quality variant panel](../img/data-quality-variant-panel.png)
+
+For variant products, the data quality panel lists all attributes that need improvements to reach a higher data quality grade, whether at model or variant level.
+
+::: info
+Black attribute labels are not clickable and must be improved at upper level.
```

Comment:
Attributes based at product model level, am I right?

Reply:
yes: at product or sub-product. Should I word it differently?

Reply:
I would rephrase it, 'at upper level' isn't as clear as it could be.

Reply:
done